File: Documentation/3.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Indexes/HowTo/GetIndexMergeSuggestions.cs
using Raven.Abstractions.Indexing;
using Raven.Client.Document;

namespace Raven.Documentation.Samples.ClientApi.Commands.Indexes.HowTo
{
	public class GetIndexMergeSuggestions
	{
		private interface IFoo
		{
			#region merge_suggestions_1
			IndexMergeResults GetIndexMergeSuggestions();
			#endregion
		}

		public GetIndexMergeSuggestions()
		{
			using (var store = new DocumentStore())
			{
				#region merge_suggestions_2
				IndexMergeResults suggestions = store.DatabaseCommands.GetIndexMergeSuggestions();
				#endregion
			}
		}
	}
}

File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/commands/indexes/howto/GetIndexMergeSuggestions.java
package net.ravendb.clientapi.commands.indexes.howto;

import net.ravendb.abstractions.indexing.IndexMergeResults;
import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;


public class GetIndexMergeSuggestions {
  @SuppressWarnings("unused")
  private interface IFoo {
    //region merge_suggestions_1
    public IndexMergeResults getIndexMergeSuggestions();
    //endregion
  }

  @SuppressWarnings("unused")
  public GetIndexMergeSuggestions() throws Exception {
    try (IDocumentStore store = new DocumentStore()) {
      //region merge_suggestions_2
      IndexMergeResults suggestions = store.getDatabaseCommands().getIndexMergeSuggestions();
      //endregion
    }
  }
}
